### PR TITLE
Change the way the Cassandra location is provided.

### PR DESCRIPTION
**Pull request by pilhuhn**
The originally given way only works for metrics, but not other components.

### CODE REVIEW

File: src/main/jbake/content/docs/user/installation.adoc [hunk 1]
```diff
@@ -40,7 +49,7 @@ From standalone.xml::
 ----
 <system-properties>
```
Comment (pilhuhn) [inline src/main/jbake/content/docs/user/installation.adoc]: Do we want to mention this block at all?

File: src/main/jbake/content/docs/user/installation.adoc [hunk 2]
```diff
@@ -24,12 +24,21 @@ Fortunately, this only requires setting
...
+[source,shell]
+----
+bin/standalone.sh -Dhawkular.backend=remote
+----
+
+.Using a cassandra cluster on different hosts
 [source,shell]
 ----
-bin/standalone.sh -Dhawkular-metrics.cassandra-nodes="node_1_address,node_2_address,node_n_address" -Dhawkular-metrics.backend=remote
+expot CASSANDRA_NODES="cassandra_host1,cassandra_host2,..."
```
Comment (tsegismont) [inline src/main/jbake/content/docs/user/installation.adoc]: typo -> `export`

File: src/main/jbake/content/docs/user/installation.adoc [hunk 3]
```diff
@@ -24,12 +24,21 @@ Fortunately, this only requires setting
 a few system properties.
```
Comment (tsegismont) [inline src/main/jbake/content/docs/user/installation.adoc]: "requires setting a few environment variables" if you no longer want to talk about sys props

Comment (pilhuhn) [inline src/main/jbake/content/docs/user/installation.adoc]: > "requires setting a few environment variables" if you no longer want 
> to talk about sys props

Thx

It is not that I don't want, but that 
`-Dhawkular-metrics.cassandra-nodes` only
applies to metrics, but not to alerts or inventory.

**Pull request merged**